src/test/java/guitests/UITest.java

```diff
@@ -395,6 +399,16 @@ public void clickMenu(String... menuNames) {
         }
     }
 
+    /**
+     * Sets a text field with given text. Does not simulate clicking and typing 
+     * in the text field.
+     * @param fieldName
+     * @param text
+     */
+    public void setTextField(String fieldName, String text) {
+        ((TextField) find(fieldName)).setText(text);
```

Comment:
Was going to use `fieldId` :sob: but was afraid that it might be too specific since it is more like a JavaFX term.